### PR TITLE
[BW] Run cron checks on release

### PR DESCRIPTION
**Pull request by testableapple**

### CODE REVIEW

**Review by polqf — APPROVED** [review]

File: .github/workflows/smoke-checks.yml [hunk 1]
```diff
@@ -1,19 +1,10 @@
 name: Smoke Checks
 
 on:
-  push:
-    branches:
-      - main
-      - develop
-
```
Comment (nuno-vieira) [inline .github/workflows/smoke-checks.yml:8]: Why are we removing this?

File: .github/workflows/smoke-checks.yml [hunk 2]
```diff
@@ -1,19 +1,10 @@
...
   pull_request:
     branches:
       - '**'
 
-  release:
-    types:
-      - created
-
```
Comment (nuno-vieira) [inline .github/workflows/smoke-checks.yml:16]: Here as well?

**Review by polqf — APPROVED** [review]

File: .github/workflows/smoke-checks.yml (hunk 2 above)
Comment (testableapple) [reply .github/workflows/smoke-checks.yml:16]: Why do we need to run some checks again after we executed them on release PR, then on push to `main`, and now on "release created" 🤔

Comment (nuno-vieira) [reply .github/workflows/smoke-checks.yml:16]: What do you mean by "after we executed them on release PR"?

Comment (nuno-vieira) [reply .github/workflows/smoke-checks.yml:16]: These checks should be run on the Release PR, that is my only concern that we are sure they are still run on Release PR

Comment (nuno-vieira) [reply .github/workflows/smoke-checks.yml:16]: All clarified now 👍

**Review by nuno-vieira — APPROVED** [review]
LGTM! ✅

File: .github/workflows/smoke-checks.yml (hunk 1 above)
Comment (testableapple) [reply .github/workflows/smoke-checks.yml:8]: Discussed this with @nuno-vieira, and we decided to remove it eventually.

**Pull request merged**